docs/docs/quickstart.ipynb

```diff
@@ -249,8 +249,22 @@
   }
  ],
  "metadata": {
+  "kernelspec": {
```

Comment:
I assume this metadata doesn't really matter much

Reply:
No, as far as I am aware it does not matter and is just updated when you make changes the Jupyter Notebook. It doesn't affect how it's rendered. @shahinrostami am I correct in thinking this meta-data doesn't matter?